tycho-its/src/test/java/org/eclipse/tycho/test/baseline/BaselineMojoTest.java

```diff
@@ -74,23 +120,36 @@ public void testAddMethod() throws Exception {
 	 * @throws Exception
 	 */
 	@Test
-	public void testAddResource() throws Exception {
+	public void testAddResource() throws Throwable {
+
 		// if version is not bumped this should fail
-		Verifier verifier = buildBaselineProject("add-resource", true);
+		Verifier verifier = buildBaselineProject(true, r -> {
+			Files.writeString(r.resolve("src/NewFile.txt"), "");
+		});
 		verifyBaselineProblem(verifier, "ADDED", "RESOURCE", "NewFile.txt", "1.0.0", "1.0.1");
-		// but if we bump the version even the smallest amout it must pass
-		buildBaselineProject("add-resource-with-bump", false);
+
+		// but if we bump the version even the smallest amount it must pass
+		buildBaselineProject(false, r -> {
+			Files.writeString(r.resolve("src/NewFile.txt"), "");
+			setManifestBundleVersion(r, "1.0.1");
+			setPomArtifactVersion(r, "1.0.1");
+		});
 	}
 
 	/**
 	 * This adds a resource to the bundle
 	 *
-	 * @throws Exception
 	 */
 	@Test
-	public void testAddHeader() throws Exception {
+	public void testAddHeader() throws Throwable {
+
 		// if version is not bumped this should fail
-		Verifier verifier = buildBaselineProject("add-header", true);
+		Verifier verifier = buildBaselineProject(true, r -> {
+			replaceStringInFileWithPlatformsLineSeparator(r.resolve("META-INF/MANIFEST.MF"), "Bundle-Version: 1.0.0",
```

Comment:
This should not use string replace, see comment about generic Manifest modification method.

Reply:
That's right. 👍🏽 Done.